### DISCUSSION TITLE
Spectrum CSS

### DISCUSSION THREAD

**Discussion by melohagan** [facebook/docusaurus#7846] [Q&A]
Hey there!

I've been exploring Docusaurus, and it looks pretty cool!

I am however trying to use Adobe Spectrum CSS: https://opensource.adobe.com/spectrum-css/

I've installed the dependencies successfully:
![Screenshot 2022-07-28 at 16 09 22](https://user-images.githubusercontent.com/101575380/181572894-be867212-b94d-490f-abce-8e31fd2edc0b.png)

And my button has the style classes:
![Screenshot 2022-07-28 at 16 10 53](https://user-images.githubusercontent.com/101575380/181573228-b495f636-66ff-4d4e-9df0-0adeb958c78c.png)

![Screenshot 2022-07-28 at 16 11 40](https://user-images.githubusercontent.com/101575380/181573381-b7da9db8-32b0-4949-81cd-99628774b921.png)

But my button does not appear to be styled:
![Screenshot 2022-07-28 at 16 12 13](https://user-images.githubusercontent.com/101575380/181573504-70a5d356-bde6-4868-8107-824624b8104f.png)

Is there anything else I need to do in my Docusaurus config?

**Comment by slorber** [comment]
I don't know, never used it.

Have you followed every step of their install process?
https://opensource.adobe.com/spectrum-css/get-started.html

Also don't forget this one? `<html class="spectrum spectrum--medium spectrum--light">`

if Docusaurus does not make it easy to add this class globally, you can at least try it using devtools.
I suspect it will set design tokens as CSS variables that other underlying components will use.
